app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v3.rb

```diff
@@ -227,9 +227,24 @@ def method_missing(method_name, *args, &block)
       end
     end
 
-    def find_mac_address_on_network(nics, network, log)
+    def get_mac_address_of_nic_on_requested_vlan(args)
```

Comment:
There are advantages I agree but this allows to path arguments in any order and change them later without breaking the interface. Its a common practice.